lib/babel.js

```diff
@@ -14,17 +17,59 @@ try {
 	babel = require('babel-core');
 }
 
-var options = {
-	blacklist: hasGenerators ? ['regenerator'] : [],
-	optional: hasGenerators ? ['asyncToGenerator', 'runtime'] : ['runtime'],
-	plugins: [
-		createEspowerPlugin(babel, {
-			patterns: require('./enhance-assert').PATTERNS
-		})
-	]
-};
-
-var transpiled = babel.transformFileSync(testPath, options);
-requireFromString(transpiled.code, testPath, {
-	appendPaths: module.paths
-});
+function options() {
+	return {
+		blacklist: hasGenerators ? ['regenerator'] : [],
+		optional: hasGenerators ? ['asyncToGenerator', 'runtime'] : ['runtime'],
+		plugins: [
+			createEspowerPlugin(babel, {
+				patterns: require('./enhance-assert').PATTERNS
+			})
+		]
+	};
+}
+
+// TODO: This needs to be smarter.
+function shouldTranspile(filePath) {
```

Comment:
This shouldTranspile definitely needs to be improved. Very naive.

I'm holding off investing time on it until we have a decision what the API and CLI should look like.